code/modules/projectiles/gun_system.dm

```diff
@@ -269,7 +278,12 @@
 		to_chat(user, "[dat.Join(" ")]")
 
 /obj/item/weapon/gun/wield(mob/user)
+	if(flags_gun_features & GUN_NO_WIELDING)
+		to_chat(user, "<span class='notice'>[src] cannot be fired by hand and must be deployed.</span>")
```

Comment:
some guns can be deployable AND can fired when not deployed? Interesting

Reply:
This is a tad wierd might consider changing this tochat or the flag name